Resources/Prototypes/GameRules/roundstart.yml

```
@@ -269,6 +269,29 @@
       mindRoles:
       - MindRoleInitialInfected
 
+- type: entity
+  parent: Thief
+  id: OopsAllThieves
+  components:
+  - type: GameRule
+    minPlayers: 30
+  - type: AntagSelection
+    agentName: thief-round-end-agent-name
+    definitions:
+    - prefRoles: [ Thief ]
+      max: 10
```

I think it'd be more interesting as more of a total conversion round, where the game becomes more like a scavenger hunt than a round of traitors / normal spess.
```suggestion
      max: 1000
```